prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir

```diff
@@ -185,7 +185,6 @@
           "code" : "P"
         } ]
       },
-      "occurredDateTime" : "2028-08-08T09:28:05-06:00",
```

Comment:
It might be nice to remove this from the other `.fhir` files too. Doesn't seem like it causes any tests to break, but it might just be nice to keep that consistent across these files.